pandas/core/frame.py

```diff
@@ -4420,7 +4421,7 @@ def merge(self, right, how='inner', on=None, left_on=None, right_on=None,
                      right_index=right_index, sort=sort, suffixes=suffixes,
                      copy=copy, indicator=indicator)
 
-    def round(self, decimals=0, out=None):
+    def round(self, decimals=0, *args):
```

Comment:
Unfortunately, `numpy` passes them in as positional in `fromnumeric.py`, so if we want to hide `out`, it needs to reside in an `*args` parameter.